packages/dm-core-plugins/blueprints/generic-list/GenericListPluginConfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "CORE:Blueprint",
+  "name": "GenericListPluginConfig",
+  "attributes": [
+    {
+      "attributeType": "boolean",
+      "type": "CORE:BlueprintAttribute",
+      "name": "collapsable",
```

Comment:
Are we sure we want to merge Accordion and List into one Plugin? They don't seem very related, except that we use them quite the same way.

Reply:
Agreed. Removed config option. Only get collapsible ones now.